scripts/integration/docker-compose.datadog-agent.yml

```diff
@@ -29,6 +30,7 @@ services:
       - DD_HEALTH_PORT=8183
       - DD_CMD_PORT=5002
       - DD_USE_DOGSTATSD=false
+      - DD_HOSTNAME=localhost
```

Comment:
:thinking: this one is pinned to 7.31 ... but I guess adding it doesn't hurt ?

Reply:
Yeah, I noticed that too and made a note to figure out why that is. I could see it coming back to bite us.